libmcount/dynamic.c

```diff
@@ -643,6 +643,74 @@ bool mcount_add_badsym(struct mcount_dynamic_info *mdi, unsigned long callsite,
 	return true;
 }
 
+void mcount_handle_dlopen(struct symtabs *symtabs, struct dl_phdr_info *info,
+			  char *mod_realpath, size_t size)
+{
+	struct uftrace_mmap *map;
+	struct mcount_dynamic_info *mdi;
+	char *libname, *modname;
+	unsigned long vaddr = 0, text_addr = 0;
+	unsigned int text_size = 0, namelen, i;
+	bool text_section_found = false;
+
+	modname = basename(mod_realpath);
+
+	if (!find_module_list(modname))
+		return;
+
+	libname = xstrdup(modname);
+	namelen = ALIGN(strlen(libname) + 1, 4);
```

Comment:
Now I think we don't need this `ALIGN(len, 4)` part.

Reply:
then ok, it was removed.